npm/private/npm_import.bzl

```diff
@@ -556,6 +557,16 @@ def _impl_links(rctx):
     for dep in ref_deps.keys():
         ref_deps[dep] = ",".join(ref_deps[dep])
 
+    lifecycle_hooks_env_list = rctx.attr.lifecycle_hooks_env
+    lifecycle_hooks_env = {}
+    for env in lifecycle_hooks_env_list:
+        key_value = env.split("=", 1)
+        if len(key_value) == 2:
+            lifecycle_hooks_env[key_value[0]] = key_value[1]
+        else:
+            msg = "lifecycle_hooks_env contains invalid key value pair %s, = not found" % env
```

Comment:
nit: if the quote the %s it will be easier to read the error if the user passes an empty string:
```
"lifecycle_hooks_env contains invalid key value pair '%s', required '=' separator not found"
```

Reply:
Got it, thank you for the tips! Updated.